### ISSUE TITLE
using "importer.DiMuMappingUpdater" throws an error

### ISSUE THREAD

**Issue by Ambrosiani** [NordicMuseum/Wikimedia-Commons-uploads#12]
I had to change `import importer.DiMuMappingUpdater as mapping_updater` to `import DiMuMappingUpdater as mapping_updater` to avoid throwing an error. According to [lokal-profil](https://github.com/lokal-profil) the code should work. This needs doublechecking/resolving so it works consistently for all users.

**Comment by Vesihiisi** [comment]
It's been a couple months so who knows what happens now.

**Comment by lokal-profil** [comment]
So `import importer.DiMuMappingUpdater as mapping_updater` is the Python 2 way of doing things. The Python 3 equivalent seems to be `import .DiMuMappingUpdater as mapping_updater` (note the `.`).

The first way should work in both Python 2 and Python 3 (based on `BatchUploadTools` working)

Per http://python-future.org/compatible_idioms.html#imports-relative-to-a-package `import DiMuMappingUpdater as mapping_updater` should not work in Python 3. (but might if you are standing in the import dir when running the script.)

**Comment by Ambrosiani** [comment]
`import .DiMuMappingUpdater as mapping_updater` works for me

**Comment by Vesihiisi** [comment]
@Ambrosiani – does this still affect you? I don't want to go all 'works on my machine' so let's try looking into it :|

**Comment by Ambrosiani** [comment]
@Vesihiisi yup, it breaks when I change it. Probably some weird, self-inflicted setting/config in my python setup so the best way forward might be to do either a step-by-step "install python 3 in this specific way" readme or putting everything in a docker container so we know exactly what's going on…

**Comment by Vesihiisi** [comment]
this is a mess and i can't reproduce it

**Comment by Ambrosiani** [comment]
I will be installing python + script on a new computer tomorrow and will see what happens :)
